aspnetcore/fundamentals/websockets.md

```diff
@@ -116,7 +116,7 @@ If using the WebSocket support in [socket.io](https://socket.io/) on [Node.js](h
 
 ## Next steps
 
-The [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/websockets/sample) that accompanies this article is an echo app. It has a web page that makes WebSocket connections, and the server resends any messages it receives back to the client. Run the app from a command prompt (it's not set up to run from Visual Studio with IIS Express) and navigate to http://localhost:5000. The web page shows the connection status in the upper left:
+The [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/websockets/samples) that accompanies this article is an echo app. It has a web page that makes WebSocket connections, and the server resends any messages it receives back to the client. Run the app from a command prompt (it's not set up to run from Visual Studio with IIS Express) and navigate to http://localhost:5000. The web page shows the connection status in the upper left:
```

Comment:
You still haven't versioned the code snippets in this file. See remark at https://github.com/aspnet/Docs/pull/8890#discussion_r223223685

Reply:
I didn't find a code snippet that is needs a versioning?

Reply:
Seems I got what you mean, updating the PR now ..

Reply:
Seems three are few other places need to be updated, have a look to the latest update